src/Panel.vue

```diff
@@ -215,6 +221,20 @@
         } else {
           return onHeaderHover;
         }
+      },
+      caretHtml () {
+        if (this.localExpanded) {
+          return '<span class="glyphicon glyphicon-chevron-down" aria-hidden="true"></span>'
+        } else {
+          return '<span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>'
+        }
+      },
+      hasCustomHeader () {
+        if (this.$slots.header) {
```

Comment:
Can this be simplified to `return this.$slots.header;`?

Reply:
Then it will return either a `null` object or a `v-node` object, which is not a `Boolean`

Reply:
How about `return this.$slots.header !== null;` then

Reply:
I was wrong, it gives `undefined` instead of `null`. Will modify accordingly.